web/vtctld2/src/app/dashboard/shard.component.ts

```diff
@@ -72,20 +65,55 @@ export class ShardComponent implements OnInit, OnDestroy {
     });
   }
 
+  getActions() {
+    let result: MenuItem[] = [
+      {label: 'Status', items: [
+        {label: 'Validate', command: (event) => {this.openValidateShardDialog(); }},
+        {label: 'Validate Version', command: (event) => {this.openValidateVerShardDialog(); }},
+        {label: 'Shard Replication Positions', command: (event) => {this.openShardReplicationPosDialog(); }},
+      ]},
+      {label: 'Change', items: [
+        {label: 'Delete', command: (event) => {this.openDeleteShardDialog(); }},
+        {label: 'Tablet Externally Reparented', command: (event) => {this.openTabExtRepDialog(); }},
```

Comment:
The reparent actions should also be on a tablet then. Note I also had moved it, but then moved it back, as I think all reparents should work on the shard. I think it's either:
- all reparents on the shard, with a dialog that has a dropdown with which tablet to use.
- all reparents on the master-elect tablet, with no tablet drop-down.

If you all think it should be on the tablet, I can move them, but I'll do it in a different PR.